src/App/AppContainer.js

```diff
@@ -70,15 +73,16 @@ class AppContainer extends Component {
 
         getAllSites.then(sites => {
             resourceSheetId = sheetIdFromPath(sites, resourcePath) || envSheetId;
-        
+
             if(resourceSheetId == null){
-                alert("Error: Unable to find resource '" + resourcePath + "'");
+				// alert("Error: Unable to find resource '" + resourcePath + "'");
+				this.setState({isValidRoute: false})
```

Comment:
also just remember to end all the JS parts with semicolons, even though it still works